Packages/Tracking/Physical Hands/Runtime/Scripts/UI/PhysicalHandsButton.cs

```diff
@@ -333,13 +347,27 @@ private void SetUpSpringJoint()
         private void FixedUpdate()
         {
             float distance = Mathf.Abs(_pressableObject.transform.localPosition.y - _initialButtonPosition.y);
+            bool isPressedByPrimaryHover = false;
 
             // Check if the button should be pressed
             if (!_isButtonPressed && distance >= (_buttonTravelDistanceLocal * 0.95f) &&
                 (_contactHandPressing || (_canBePressedByObjects && _objectsContactingButton.Count > 0)))
             {
-                _isButtonPressed = true;
-                ButtonPressed();
+                if (_usePrimaryHover)
+                {
+                    foreach (var hand in _contactedHands)
+                    {
+                        if (_primaryHovered.ContainsKey(hand.Key) && _primaryHovered[hand.Key] == true)
```

Comment:
can we avoid using containskey every frame?

perhaps just have 2 bools, one for each hand being primary hovered, and check if the one that matches the chirality of "hand" is true?